app/src/main/java/openfoodfacts/github/scrachx/openfood/views/BottomNavigationBehavior.java

```diff
@@ -0,0 +1,59 @@
+package openfoodfacts.github.scrachx.openfood.views;
+
+import android.support.annotation.NonNull;
+import android.support.design.widget.BottomNavigationView;
+import android.support.design.widget.CoordinatorLayout;
+import android.support.v4.view.ViewCompat;
+import android.view.View;
+
+/**
+ * Created by IMLV on 17-03-2018.
+ */
+
```

Comment:
You can also use the @author and @version tags here, although I can see this is generated documentation

Reply:
@ross-holloway94 this is auto-generated when we create a new class so IMO we should not modify this, it generates in the same way everywhere when a new class is created ..